schema/spacewalk/upgrade/susemanager-schema-5.1.6-to-susemanager-schema-5.1.7/001-update-system-overview.sql

```diff
@@ -0,0 +1,368 @@
+--
```

Comment:
wrong schema dir. Should be now in `susemanager-schema-5.1.7-to-susemanager-schema-5.1.8`